src/css/scss/chatbot-widget.scss

```diff
@@ -190,6 +190,7 @@ $text-color-xlight: $white;
 
         .a-button {
             display: inline-block;
+            margin-right: $spacer-xs / 2;
```

Comment:
Vreemd dat hier geen spatie wordt gezet. Misschien float die button en kan er beter `float: none` gedefinieerd worden?

Reply:
Ik kan me niet herinneren dat de knop aan het floaten was. Ik kan het momenteel niet meer reproduceren omdat de chatbot waarop ik het testte weggedaan is. Ik stel voor van het zo te laten.